src/compiler.ts

```diff
@@ -1827,6 +1827,7 @@ export class Compiler extends DiagnosticEmitter {
     var tableBase = this.options.tableBase;
     if (!tableBase) tableBase = 1; // leave first elem blank
     index = tableBase + functionTable.length;
+    assert(index >= 1 << 31)
```

Comment:
I guess it should be `assert(index <= ((1 << 31) >>> 0));`

Reply:
oh right, whoops.